package.json

```diff
@@ -1,12 +1,56 @@
 {
+  "version": "0.4.0",
+  "build": {
+    "publish": [
+      "github"
+    ],
+    "appId": "TinyTacoTeam.ZazuApp",
```

Comment:
`productName` is missing here. Previously, the `productName` is `Zazu`, and without it, the name become `zazu`, which are all lower case. This affects some other fields in the generated app, such as `CFBundleDisplayName`:

Previously, the `Info.plist` contains:

```xml
        <key>CFBundleDisplayName</key>
        <string>Zazu</string>
        <key>CFBundleExecutable</key>
        <string>Zazu</string>
```

And now, it becomes:

```xml
    <key>CFBundleDisplayName</key>
    <string>zazu</string>
    <key>CFBundleExecutable</key>
    <string>zazu</string>
```